ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -372,6 +373,11 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   };
 
   const isStackedBar = visual.display === 'bar' && visual.stack === 'All';
+  const tooltipConfig: TooltipConfig = {
+    ...DEFAULT_TOOLTIP_CONFIG,
+    enablePinning: tooltip?.enable_pinning ?? true,
+    enableEChartsTooltip: isStackedBar,
```

Comment:
It might be worth adding a comment here explaining why we use the native echarts tooltip for stacked bars. I'm personally curious why since I missed those code changes while I was out on vacation, and I could see other folks being curious in the future when looking at the code.

Reply:
So the custom tooltip didn’t always reflect the subgroup you were hovering over accurately, there’s a way to fix it but just using the ECharts tooltip in this case was less work and since you shouldn’t really use that chart type with a ton of series—the nearby-series / highlighting logic isn’t as important